cloudlift/deployment/service_template_generator.py

```diff
@@ -959,6 +981,71 @@ def _get_desired_task_count_for_service(self, service_name):
             return self.desired_counts[service_name]
         else:
             return 0
+    def _firelens_container_def_args_override(self, configuration, container_def_args):
+        if configuration.get('logging') == 'awsfirelens':
+            container_def_args['DependsOn'] = container_def_args.get('DependsOn', []) + [ContainerDependency(ContainerName='fluentbit-firelens-sidecar', Condition='START')]
+        return container_def_args
+
+    def _sidecar_container_defs(self, configuration, service_name):
+        container_definitions = []
+        if configuration.get('sidecars') and isinstance(configuration['sidecars'], list):
+            for index, sidecar in enumerate(configuration['sidecars']):
+                container_name = sidecar.get('name', f"sidecar_{index + 1}")
+                if not container_name.endswith('-sidecar'):
+                    container_name = container_name + '-sidecar'
+                image_uri = sidecar.get('image_uri')
+                memory_reservation = int(sidecar.get('memory_reservation', 50))
+                essential = sidecar.get('essential', True)
+                
+                mount_points = []
+                if sidecar.get('mount_points') and isinstance(sidecar['mount_points'], list):
```

Comment:
Let review if we need this  @rokibulislaam

Reply:
removed `mount_points`